### ISSUE TITLE
Pelles C's Windows API headers

### ISSUE THREAD

**Issue by 3oheicrw** [LADSoft/OrangeC#721]
I have the same idea as https://github.com/LADSoft/OrangeC/issues/715 except to avoid any legal issues, I think support for this should be implemented in OrangeC itself instead of modifying the Pelles C headers. OrangeC will have to emulate `__POCC__` I think. The idea is the same as Chromium people using `clang-cl` to build on Windows, it's `clang` but take advantage of MSVC headers.

**Comment by LADSoft** [comment]
well i'd actually heard enough people grumbling about this to at least look into the pelles headers...

pelles licensing is ok, they just don't want you selling it.   Which is fine OCC is not a commercial product lol!

So I played with the headers for about half an hour and aside from a couple of #defines that have to go into the compiler it is pretty clean...   there are some bugs in the headers though.   Still, I got a simple include of windows.h down to 1 error and a handful of warnings in a little under an hour with no compiler modifications...   so I think I'm planning to do this now.

stay tuned!
